src/Components/Footer/Footer.jsx

```diff
@@ -14,7 +14,7 @@ export const Footer = () => {
                 <p className="email-username">{userEmail}</p>
             </div>   
             <div className="flex flex-column pdl-2">
-                <button className="btn-logout btn-transparent" onClick={() => logout()}><span class="material-icons-outlined text-3xl">logout</span></button>
```

Comment:
standard editor config, maxLineLength is 80 chars
reference: https://editorconfig.org/

Reply:
Thanks for providing the link.